packages/core/src/test/TestContext.ts

```diff
@@ -540,6 +567,11 @@ export class TestContext {
     /** Override trace completely. */
     finalImports.env.trace = this.trace.bind(this);
 
+    /** Add support for wasi if requested. */
+    if (this.wasi) {
+      finalImports.wasi_snapshot_preview1 = this.wasi.wasiImports;
```

Comment:
```suggestion
      finalImports.wasi_snapshot_preview1 = this.wasi.wasiImport;
```

I was just testing this branch with Node 13.11.0 and it was breaking on this line. Looks like this should be `wasiImport`, not `wasiImports`.

Reply:
Thanks for the help. I'm not entirely sure how this should be tested, to be honest. What kind of AssemblyScript code would be appropriate to validate that WASI is functioning?

Reply:
No problem! I actually don't have any formal tests or anything. I just created a new test that imports `Console` from `as-wasi`, and then manually checked to see that `Console.log('test')` worked.